docs/Questions.md

```diff
@@ -0,0 +1,98 @@
+<H1> Folder structure(Pages, Components, Templates, Organisms, Molecules, Atoms) </H1>
+We should stick to one way of organizing our folder and component's structure.
+Utils have files with .tsx file extension, which I think is inappropriate.
+We have validation.js and wdyr.js files at the top level of src
+
+<H1> Styles </H1>
+Currently we both use Styled-Components and scss across our codebase. I think we should stick to one and remove the other
```

Comment:
Personally, I prefer styled-components because they allow you to pass variables to your styles from the component

Reply:
In most cases I have seen `styled-components` used, it seems like a heavyweight solution for things we can already do quite easily with scss/etc. It was introduced into the codebase with some of @IvanBuljovcic's work from memory, but I wasn't really a fan of us following that pattern at the time, and I still don't think it's a great choice for our future patterns.

I would be interested to know what specific issues/cases `styled-components` makes things better for us/'solves a problem' that isn't easily possible with standard scss/etc, as every case I have seen it used, I believe there is a better 'standard solution'.

Semi-related, I started capturing some of my thoughts around improving our current styles/related patterns in https://github.com/sparkletown/internal-sparkle-issues/issues/4 It's definitely not fully fleshed out, more a starting point.